src/els_db.erl

```diff
@@ -16,7 +16,7 @@
                 , els_dt_references
                 , els_dt_signatures
                 ]).
--define(TIMEOUT, 5000).
```

Comment:
This might need to be even bigger. I bumped it to 20 seconds locally since I was still getting timeouts.

Reply:
Bumped to 20s for now.